services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js

```diff
@@ -1,16 +1,80 @@
 import React from "react";
-import ChipInput from "material-ui-chip-input";
-import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
-import {Button, Dialog, DialogActions, DialogContent, DialogTitle} from "@material-ui/core";
+import {Profile} from "utils/models";
+
+import {
+    Button,
+    Chip,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    IconButton,
+    TextField
+} from "@material-ui/core";
+import AddIcon from "@material-ui/icons/Add";
 import "./EditSkillsDialog.scss";
+import classNames from "classnames";
 
 const EditSkillsDialog = ({databaseSkills, skills, handleCancel, handleSubmit, open}) => {
+    //Sort the Database skills and
```

Comment:
...

Reply:
I will make this change in the remove ticket